### PR TITLE
Sophisticated form templates for issues 

### PR DESCRIPTION
**Pull request by gkarthiks**
Signed-off-by: gkarthiks <github.gkarthiks@gmail.com>

#### What this PR does / why we need it:

This PR replaces the existing markdown issue template with sophisticated form templates.

#### Special notes for your reviewer:

Please see the screenshots below.
### Bug Report

image

### Feature Request
image

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      required: true
+
+
+  - type: textarea
+    id: after-values
+    attributes:
+      label: Enter the Helm values set after installation/upgrade.
+      description: Pate the output of the command `helm get values my-release`
+    validations:
+      required: false
```
Comment (torstenwalter) [inline .github/ISSUE_TEMPLATE/bug_report.yml:100]: do we really need it? I mean we have `changed-values` section above it should be sufficient to reproduce the issue. If people are too lazy to have a minimal example to be able to reproduce it then I am asking myself if I should invest the time to figure out how to reproduce it.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      placeholder: 'key: value'
+    validations:
+      required: false
+
+  - type: input
+    id: helm-command
+    attributes:
+      label: Enter the command that you execute and failing/misfunctioning.
+      description: Enter the command as-is as how you executed.
+      placeholder: helm install my-release prometheus-community/name-of-chart --version version --values values.yaml
```
Comment (torstenwalter) [inline .github/ISSUE_TEMPLATE/bug_report.yml:89]: The placeholder is not completely visible:
image

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    validations:
+      required: true
+
+
```
Comment (torstenwalter) [inline .github/ISSUE_TEMPLATE/bug_report.yml:93]:
```suggestion

```

**Review by torstenwalter — COMMENTED** [review]
I really like the proposal. Have not seen these forms before!

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (gkarthiks) [reply .github/ISSUE_TEMPLATE/bug_report.yml:100]: I agree. I will remove that. 

I was just porting over our markdown prompts to these forms ;)

Comment (gkarthiks) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: done. removed the `after-values` prompt.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (gkarthiks) [reply .github/ISSUE_TEMPLATE/bug_report.yml:89]: made the input box to textarea. It now displays as below.

image

Comment (gkarthiks) [reply .github/ISSUE_TEMPLATE/bug_report.yml:89]: @torstenwalter if this looks good, please resolve and merge!!! :pray:

**Review by torstenwalter — APPROVED** [review]

**Comment by torstenwalter** [comment]
Thanks a lot @gkarthiks. This looks really nice :100: 

Could we do something similar for the pull request template?

**Comment by gkarthiks** [comment]
Sure will do it.

**Pull request merged**